ruleset/sca/windows/cis_win10_enterprise.yml

```
@@ -847,13 +883,14 @@ checks:
     remediation: "To establish the recommended configuration via GP, set the following UI path to Classic - local users authenticate as themselves: Computer Configuration\\Policies\\Windows Settings\\Security Settings\\Local Policies\\Security Options\\Network access: Sharing and security model for local accounts."
     compliance:
       - cis: ["2.3.10.12"]
-      - cis_csc: ["14", "16"]
       - pci_dss: ["7.1.3"]
       - tsc: ["CC6.4"]
     references:
-      - 'CCE-33719-6'
-    condition: all
+      - "CCE-33719-6"
+    condition: any
     rules:
+      - 'r:HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Lsa'
+      - 'r:HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Lsa -> ForceGuest'
```

Using condition 'any' probably means this should be:
```suggestion
      - 'not r:HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Lsa -> ForceGuest'
```

---

Done 18dbade